Packs/CommonPlaybooks/Playbooks/playbook-Search_For_Hash_In_Sandbox_-_Generic.yml

```diff
@@ -1176,8 +970,260 @@ tasks:
+    nexttasks:
+      '#none#':
+      - "44"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2040,
+          "y": 1550
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 52e49580-6f9b-430f-81a4-a98dfe2b2813
+    type: regular
+    task:
+      id: 52e49580-6f9b-430f-81a4-a98dfe2b2813
+      version: -1
+      name: Set Detected Hashes - WildFire
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/playbooks/automations.html
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "37"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: DetectedHashes
+      value:
+        complex:
+          root: WildFire.Report
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: WildFire.Report.Status
+                iscontext: true
+              right:
+                value:
+                  simple: Success
+              ignorecase: true
+          accessor: SHA256
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2040,
+          "y": 1030
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 668a18cd-a176-4315-8bc1-0ac660fe91ef
+    type: regular
+    task:
+      id: 668a18cd-a176-4315-8bc1-0ac660fe91ef
+      version: -1
+      name: Set Detected Hashes - Falcon Intelligence Sandbox
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-admin/playbooks/automations.html
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "37"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: DetectedHashes
```

Comment:
let's add this key to the PB outputs as well

Reply:
Done - added the 'DetectedHashes' context key to the playbook outputs.